custom/icds_reports/filters.py

```diff
@@ -105,6 +105,10 @@ class IcdsRestrictedLocationFilter(AsyncLocationFilter):
     def location_hierarchy_config(self):
         return location_hierarchy_config(self.domain)
 
+    def load_locations_json(self, loc_id):
+        show_test = self.request.GET.get('include_test', False)
+        return load_locs_json(self.domain, loc_id, user=self.request.couch_user, show_test=show_test)
+
```

Comment:
Do we need two separate filters `IcdsLocationFilter` and `IcdsRestrictedLocationFilter` ?
They are now the same with the only difference that `IcdsRestrictedLocationFilter` uses a limited location hierarchy. I guess we could just keep one but I am not sure, just floating the idea. 
Feel free to punt that if that does not look obvious/straight forward/unnecessary

Reply:
`IcdsRestrictedLocationFilter` could just inherit from `IcdsLocationFilter`

Reply:
thanks for that idea. Let me check

Reply:
Make it inherit the MPR location filter class